docs/README.md

```diff
@@ -86,7 +86,7 @@ Simply update the associated `.md` markdown file. All documentation should be in
 
 # Testing changes
 
-If you have Jekyll stack installed you can run the website locally to test changes. To install Jekyll, follow the [Setting up your GitHub Pages site locally with Jekyll](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/) tutorial.  
```

Comment:
Me again, just one more nit...🌈 Something looks weird here. Are you sure about this? 🌈